### DISCUSSION TITLE
PHP Database

### DISCUSSION THREAD

**Discussion by akashdip2001** [community/community#34542] [Other Feature Feedback, Questions, & Ideas]
Please add Dynamic web page option in Github pages

**Comment by jfullstackdev** [comment]
GitHub Pages was actually designed so that full documentation can be achieved, and documentation does not require server-side scripting

but who knows they might implement this in the future but this will really be heavy for the `github.io` domain or might give this not for free,

**Comment by SilasRodrigues19** [comment]
@akashdip2001 U can use [InfinityFree](https://www.infinityfree.net/) which has support for PHP and MySQL and is for free. It doesn't have integration with GitHub, but u just have to host ur code there too.
